src/launch/policy.c

```diff
@@ -533,6 +559,18 @@ static int policy_import_recv(Policy *policy, ConfigNode *cnode) {
                         return error_trace(r);
 
                 c_list_link_tail(&node->recv_list, &record->link);
+        } else if (cnode->parent->policy.context == CONFIG_POLICY_NO_CONSOLE) {
+                r = policy_at_uid(policy, &node, 0, strtoul(SYSTEMUIDMAX, NULL, 10));
+                if (r)
+                        return error_trace(r);
+
+                c_list_link_tail(&node->recv_list, &record->link);
+        } else if (cnode->parent->policy.context == CONFIG_POLICY_AT_CONSOLE) {
+                r = policy_at_uid(policy, &node, strtoul(SYSTEMUIDMAX, NULL, 10) + 1, -1);
```

Comment:
With `SYSTEMUIDMAX` being an integer, you should be able to drop all these `strtoul` calls.

Also, how about `policy_at_systemuid()` as a small helper that hard-codes 0-SYSTEMUIDMAX?

Reply:
Yup. Done.